viewing_party/party.py

```diff
@@ -0,0 +1,135 @@
+def create_movie(title, genre, rating):
+    new_movie = {
+        "title" : title,
+        "genre" : genre,
+        "rating" : rating,
+    }
+
+    # != and "is not" do the same
+    if title != None and \
+        genre != None and \
+        rating != None:
+        return new_movie
```

Comment:
`==` is an equality test (it checks to see if both sides are equal objects) and `is` is an identity test (checks to see if both sides are the exact same object)